app/tasks/specs.php

```diff
@@ -0,0 +1,271 @@
+<?php
+
+global $cli;
+
+use Utopia\Validator\Text;
+use Appwrite\Specification\Format\OpenAPI3;
+use Appwrite\Specification\Format\Swagger2;
+use Appwrite\Specification\Specification;
+use Appwrite\Utopia\Response;
+use Swoole\Http\Response as HttpResponse;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+use Utopia\Request;
+use Utopia\Validator\WhiteList;
+
+$cli
+    ->task('specs')
+    ->param('version', 'latest', new Text(8), 'Spec version', true)
+    ->param('mode', 'normal', new WhiteList(['normal', 'tests']), 'Spec Mode', true)
+    ->action(function ($version, $mode) use ($register) {
+        $db = $register->get('db');
+        $redis = $register->get('cache');
+        $appRoutes = App::getRoutes();
+        $response = new Response(new HttpResponse());
+        $tests = ($mode === 'tests');
+        
+        App::setResource('request', function() {
+            return new Request;
+        });
+
+        App::setResource('response', function() use ($response) {
+            return $response;
+        });
```

Comment:
```suggestion
        App::setResource('request', fn() => new Request);
        App::setResource('response', fn() => $response);
```

we can be fancy here now :-)